packages/configure-splash-screen/src/constants.ts

```diff
@@ -27,3 +33,22 @@ export interface Parameters {
    */
   darkModeImagePath?: string;
 }
+
+/**
+ * These might be optionally provided by the user. There are default values for them.
+ */
+export interface StatusBarOptions {
+  statusBarHidden: boolean;
+  statusBarStyle: StatusBarStyle;
+}
```

Comment:
If they are optional (as said in the comment), shouldn't it be `Partial`?

Reply:
Not really, the second part of comment explains that there are always some value for each of these 😉